config/schedule.yml

```diff
@@ -0,0 +1,3 @@
+log_worker_queue_stats:
+  cron: "*/10 * * * *"
```

Comment:
This is every 10 minute?

Reply:
Yep! Matches our scheduler 
![Screen Shot 2020-08-14 at 2 43 40 PM](https://user-images.githubusercontent.com/1813380/90287007-95273480-de3c-11ea-90a3-8caf8d25f64a.png)